pom.xml

```diff
@@ -89,6 +89,11 @@
       <artifactId>jansi</artifactId>
       <version>1.9</version>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-aggregator</artifactId>
+      <version>1.8</version>
```

Comment:
This should be a `test`-scoped dependency. (Or deleted, if you do not plan to add an actual `JenkinsRule`-based test verifying Workflow compatibility.)